coupon_criteria_multi_product/migrations/15.0.1.0.0/pre-migration.py

```diff
@@ -9,7 +9,18 @@
 _table_renames = [("sale_coupon_criteria", "coupon_criteria")]
 
 
+def install_new_modules(env):
```

Comment:
Private method?? _install_new_modules...

Reply:
What for? Private methods only work at RPC level